### PR TITLE
[fix](outfile) fix outfile csv did not write json column with string

### PR DESCRIPTION
**Pull request by nextdreamblue**
## Proposed changes

Issue Number: close #xxx

before this pr, outfile or export by csv,  json type column will write json binary, not string, and if load this csv, will get error

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run p0

**Comment by nextdreamblue** [comment]
run P0 Regression PipelineX

**Comment by nextdreamblue** [comment]
run PipelineX

**Comment by nextdreamblue** [comment]
run p0PipelineX

**Comment by nextdreamblue** [comment]
run p0pipelinex

**Comment by nextdreamblue** [comment]
run p0x

**Comment by nextdreamblue** [comment]
run pipelinex_p0

**Comment by nextdreamblue** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run external

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by morningman** [comment]
@BePPPower PTAL

File: be/src/vec/data_types/serde/data_type_jsonb_serde.cpp [hunk 1]
```diff
@@ -83,7 +83,10 @@ Status DataTypeJsonbSerDe::serialize_one_cell_to_json(const IColumn& column, int
 
     const StringRef& s = assert_cast<const ColumnString&>(*ptr).get_data_at(row_num);
     if (s.size > 0) {
-        bw.write(s.data, s.size);
+        std::string str = JsonbToJson::jsonb_to_json_string(s.data, s.size);
+        bw.write(str.c_str(), str.size());
+    } else {
+        bw.write("\\N", 2);
```
Comment (BePPPower) [inline be/src/vec/data_types/serde/data_type_jsonb_serde.cpp:89]: You can use `NULL_IN_CSV_FOR_ORDINARY_TYPE`, rather than `\\N`

**Comment by nextdreamblue** [comment]
run buildall

File: be/src/vec/data_types/serde/data_type_jsonb_serde.cpp (hunk 1 above)
Comment (nextdreamblue) [reply be/src/vec/data_types/serde/data_type_jsonb_serde.cpp]: done

**Comment by BePPPower** [comment]
Please fix BE code formatter.

**Comment by nextdreamblue** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by nextdreamblue** [comment]
run pipelinex_p0

**Comment by nextdreamblue** [comment]
run buildall

**Review by cambyzju — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]

**Pull request merged**